dotNETStandard/HealthVaultProviderManagementPortal/HealthVaultProviderManagementPortal/Views/Patient/Index.cshtml

```diff
@@ -0,0 +1,69 @@
+
+    The above copyright notice and this permission notice shall be included in all copies or substantial portions of the Software.
+
+    THE SOFTWARE IS PROVIDED *AS IS*, WITHOUT WARRANTY OF ANY KIND, EXPRESS OR IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY, FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM, OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE SOFTWARE.
+*@
+@using HealthVaultProviderManagementPortal.Models.Patient
+@model HealthVaultProviderManagementPortal.Models.Patient.TimelineViewModel
+
+@{
+    ViewBag.Title = "Patient preview";
+    DateTime date = DateTime.MinValue;
+}
+
+<h1 class="page-title">Patient preview</h1>
+
+@using (Html.BeginForm("Index", "Patient", FormMethod.Get))
+{
+    @Html.Hidden("personID", Request.Params["personId"])
+    @Html.Hidden("recordID", Request.Params["recordID"])
+
+    <div class="form-horizontal">
+        <div class="form-group">
+            <div class="col-md-5 slim-col">
+                @Html.LabelFor(model => model.StartDate, htmlAttributes: new { @class = "control-label" })
+                @Html.EditorFor(model => model.StartDate, new { htmlAttributes = new { @class = "form-control" } })
+            </div>
+            <div class="col-md-5 slim-col">
+                @Html.LabelFor(model => model.EndDate, htmlAttributes: new { @class = "control-label" })
+                @Html.EditorFor(model => model.EndDate, new { htmlAttributes = new { @class = "form-control" } })
+            </div>
+            <div class="col-md-2 slim-col">
+                <input type="submit" value="Get" class="btn btn-default btn-inline" />
+            </div>
+        </div>
+    </div>
+}
+
+@if (Model?.TimelineEntries != null && Model.TimelineEntries.Any())
+{
+    <h3>Timeline</h3>
+
+    foreach (var entry in Model.TimelineEntries)
+    {
+        <h4>@entry.Key</h4>
+
+        <table class="table table-striped">
+            <tbody>
+                @foreach (var task in entry)
+                {
+                    <tr>
+                        <td class="ms-thumbnail"><img class="img-thumbnail" src="@task.TaskImageUrl" alt="Task thumbnail icon" /></td>
+                        <td>
+                            @task.TaskName<br />
+                            @(task.ScheduleType == TimelineScheduleType.Anytime ?
+                                TimelineScheduleType.Anytime.ToString() :
+                                task.LocalDateTime.TimeOfDay.ToString())
+                        </td>
+                        <td><!--@task.TaskId--></td>
```

Comment:
I left it there for debugging for now. Relatively soon, I'll be adding a link to record an occurrence, which will use this ID anyways.